registration/templates/registration/_base/navigation.html

```diff
@@ -0,0 +1,35 @@
+{% load i18n %}
+
+{% for troop in request.user.troops.all %}
```

Comment:
What about higher level staff that has access to multiple scout troops? Since most users will only have one troop to manage, directly accessing it via the navigation bar is nice. But I think we should have an alternative if a user manages more than two troops.

Reply:
> What about higher level staff that has access to multiple scout troops?

I think that most people will have access to 1 troop, and some people maybe 2 or 3 (their own Troop + Bezirk for instance). In this case the menu will simply be repeated (so that they have a clear view of who is registered where, and how much money will be owed from which troop).

![image](https://user-images.githubusercontent.com/3864879/75018736-e1964680-548f-11ea-8b69-3341589e6a98.png)

For very-high level staff, they should not use those pages, but the pages listed further below in the menu (under "Verwaltung" for instance): they will need search function, filters and export features (which are probably useless for "lower-level staff")

Reply:
Ok, so let's just add views and menu items for administration later on.